day2/exercise1/solution.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	words := [5]string {"quick","brown","fox","lazy","dog"}
+	var state = make(map[string]int)
+
+	charChannel := make(chan string)
+
+	go func() {
+		for{
+			char,done:=<-charChannel
+			if done{
+				state[char]++
+			} else{
+				return
+			}
+
+		}
+	}()
+
+	for _,word := range words{
+		for _,char := range word{
```

Comment:
you have used nested for loops making it linear. No advantage of go routines here.